src/Toolbar.tsx

```diff
@@ -158,7 +158,8 @@ export function Toolbar({
       const isDisabled =
         disableAllButtons ||
         !isConfiguredAvailable ||
-        maybeIconConfig.disabled?.(state) === true
+        maybeIconConfig.disabled?.(state) === true ||
+        kclManager.hasErrors()
```

Comment:
@franknoirot I think I'd like to wrap `hasErrors()` here in some `hasNoKclErrors(kclManager)` function or something like that so we can edit it in one place if need be